src/test/scala/nl.knaw.dans.easy.ccw/CollectSpec.scala

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (C) 2017 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.ccw
+
+import better.files.File
+import nl.knaw.dans.easy.ccw.Command.configuration
+import org.apache.commons.configuration.PropertiesConfiguration
+import org.scalatest.BeforeAndAfterAll
+
+import scala.language.postfixOps
+import scala.util.Success
+
+class CollectSpec extends TestSupportFixture with BeforeAndAfterAll {
+
+  val resourceDir = File(getClass.getResource("/"))
+  val configuration = new Configuration("version x.y.z",
+    new PropertiesConfiguration() {},
+    new PropertiesConfiguration() {
+      setDelimiterParsingDisabled(true)
+      load(resourceDir / "debug-config" / "datamanager.properties" toJava)
+    })
+  val datamanagerProperties = configuration.datamanagers
+  val commonCurationArea = testDir / "easy-common-curation-area"
+  val datamanagerCurationAreas = testDir / "datamanager-curation-areas"
+  val managerCurationDir = datamanagerCurationAreas / "$unix-user/curation-area" toString
+  def managerCurationArea(datamanager: DatamanagerId) = File(managerCurationDir.replace("$unix-user", datamanager))
+  val jannekesCurationArea = datamanagerCurationAreas / "janneke/curation-area"
+  val testLog = testDir / ".." / "test.log"
+
+  val collector = new EasyCollectCurationWorkApp(commonCurationArea, managerCurationArea, datamanagerProperties)
+
+  override def beforeAll(): Unit = {
+    testLog delete (true)
```

Comment:
I think it will be clearer if you write it like `testLog.delete(swallowIOExceptions = true)`

same in other places in this file